docs/src/components/demo/Demo.css

```diff
@@ -163,57 +172,35 @@
       }
     }
 
-    /* Scroll containers may be focusable */
-    & pre:focus-visible {
-      position: relative;
-      outline: 2px solid var(--color-blue);
-      outline-offset: -1px;
-      z-index: 1;
-    }
-
-    &::before {
-      content: '';
-      position: absolute;
-      pointer-events: none;
-      height: 7.5rem;
-      bottom: 0;
-      left: 0;
-      right: 0;
-      background: linear-gradient(to bottom, rgb(255 255 255 / 0), rgb(255 255 255 / 60%));
-    }
-    @media (prefers-color-scheme: dark) {
-      &::before {
-        background: linear-gradient(to bottom, rgb(0 0 0 / 0), rgb(0 0 0 / 60%));
-      }
+    & code {
+      /* Different fonts may introduce vertical align issues */
+      display: block;
+      /* Make sure selection highlight spans full container width in Safari */
+      flex-grow: 1;
     }
 
-    &[data-open] {
+    &[data-closed] {
```

Comment:
We didn't have the closed state before iirc, much easier to style now. Not undoing any styles anymore